lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -167,44 +168,47 @@ private void processErrorAsyncCriResponse(ErrorAsyncCriResponse errorAsyncCriRes
                         .with(
                                 LOG_ERROR_DESCRIPTION.getFieldName(),
                                 errorAsyncCriResponse.getErrorDescription())
-                        .with(LOG_ERROR_CODE.getFieldName(), errorAsyncCriResponse.getError())
-                        .with(
-                                LOG_CRI_ISSUER.getFieldName(),
-                                errorAsyncCriResponse.getCredentialIssuer()));
+                        .with(LOG_ERROR_CODE.getFieldName(), errorAsyncCriResponse.getError()));
 
-        sendIpvVcErrorAuditEvent(errorAsyncCriResponse);
+        if (criResponseItem.isPresent()
+                && Cri.F2F.equals(Cri.fromId(criResponseItem.get().getCredentialIssuer()))) {
+            sendIpvVcErrorAuditEvent(errorAsyncCriResponse);
+        }
     }
 
     private void processSuccessAsyncCriResponse(SuccessAsyncCriResponse successAsyncCriResponse)
             throws ParseException, CiPutException, AsyncVerifiableCredentialException,
                     CiPostMitigationsException, VerifiableCredentialException,
                     UnrecognisedVotException, CredentialParseException, EvcsServiceException,
                     HttpResponseExceptionWithErrorBody {
-        final CriResponseItem criResponseItem =
-                criResponseService.getCriResponseItem(
-                        successAsyncCriResponse.getUserId(),
-                        successAsyncCriResponse.getCredentialIssuer());
-        validateOAuthState(successAsyncCriResponse, criResponseItem);
+        var userId = successAsyncCriResponse.getUserId();
+        var state = successAsyncCriResponse.getOauthState();
+        Optional<CriResponseItem> criResponseItem = getCriResponseItem(userId, state);
+
+        if (criResponseItem.isEmpty()) {
+            LOGGER.error(LogHelper.buildLogMessage("No response item found"));
```

Comment:
maybe 'No response item found given user id and state'? as this is effectively replacing the state validation